projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.html

```diff
@@ -25,7 +25,7 @@
        class="pdfViewer"
        mvCreateTextHighlight
        [ngClass]="{ hidden: loadingDocument, highlightMode: highlightMode | async, drawMode: drawMode | async }">
-  </div>
```

Comment:
you cannot do this, because the content of this div is set dynamically by pdf.js
by including mv-redactions/mv-metadata-layer within the div, you've effectively removed them, because the content of the div is overwritten by pdf.js